src/OrbitGl/CaptureWindowTest.cpp

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gtest/gtest.h>
+
+#include "CaptureWindow.h"
+#include "UnitTestSlider.h"
+
+namespace orbit_gl {
+
+class UnitTestCaptureWindow : public CaptureWindow, public testing::Test {
+ public:
+  explicit UnitTestCaptureWindow() : CaptureWindow(nullptr) {
+    slider_ = std::make_unique<UnitTestSlider>(viewport_, false);
+    vertical_slider_ = std::make_unique<UnitTestSlider>(viewport_, true);
+  }
+};
+
+TEST_F(UnitTestCaptureWindow, SlidersRespondToMouseOver) {
+  Resize(100, 200);
+
+  GlSlider* slider = FindSliderUnderMouseCursor(50, 50);
+  EXPECT_EQ(nullptr, slider);
+  slider = FindSliderUnderMouseCursor(95, 10);
+  EXPECT_EQ(vertical_slider_.get(), slider);
+  slider = FindSliderUnderMouseCursor(5, 195);
+  EXPECT_EQ(slider_.get(), slider);
+  slider = FindSliderUnderMouseCursor(95, 195);
+  EXPECT_EQ(nullptr, slider);
+
+  MouseMoved(95, 10, false, false, false);
+  EXPECT_TRUE(dynamic_cast<UnitTestSlider*>(vertical_slider_.get())->IsMouseOver());
+
+  MouseMoved(5, 195, false, false, false);
+  EXPECT_FALSE(dynamic_cast<UnitTestSlider*>(vertical_slider_.get())->IsMouseOver());
+  EXPECT_TRUE(dynamic_cast<UnitTestSlider*>(slider_.get())->IsMouseOver());
+
+  MouseMoved(50, 50, false, false, false);
+  EXPECT_FALSE(dynamic_cast<UnitTestSlider*>(vertical_slider_.get())->IsMouseOver());
+  EXPECT_FALSE(dynamic_cast<UnitTestSlider*>(slider_.get())->IsMouseOver());
```

Comment:
I'm a bit hesitant to put this into GlSlider because it's really only exposing internal state that shouldn't be needed in the public API. Not a big issue, but I'm trying to keep the public API minimal.